client/webserver/site/src/css/main.scss

```diff
@@ -479,8 +479,9 @@ div.form-closer {
   background-color: #040012;
   border: 1px solid #333;
   color: white;
-  max-width: 500px;
+  max-width: 300px;
   font-size: 15px;
+  border-radius: 7.5px;
```

Comment:
This is larger than any border radius we're using anywhere else (except where we're using it to make a circle). How about `3px`?

Reply:
> This is larger than any border radius we're using anywhere else

Yup, it is. I added an image in my first comment to show what it'd look like.

> How about 3px?

Ermm, if that's what we want then np.

Reply:
We use 3 or 4 px for all of our border radii. The few places you see larger values are when we are creating circles.

Reply:
I'd use 4 px.

Reply:
Updated.